app/assets/stylesheets/base/_base.scss

```diff
@@ -320,37 +324,49 @@
   $card-margin: 1em;
   $card-image-hover-opacity: 0.7;
   $card-image-hover-color: #F8F2B4;
+  $texture-color: #FFF4EB;
+  $texture-top-background: lighten($texture-color, 1%);
+  $texture-bottom-background: darken($texture-color, 10%);
 
   @include flex-basis(15em);
   @include flex-grow(1);
   @include transition (all 0.2s ease-in-out);
+  // gradient color 1, gradient color 2, gradient angle, texture-number, 0=inverted or 1=normal  
+  @include texture(180deg, $texture-top-background, $texture-bottom-background, 16, 1); 
   background-color: $card-background;
-  border: $card-border;
   border-radius: $base-border-radius;
+  border: $card-border;
   box-shadow: 0 2px 4px darken($base-background-color, 10%);
   cursor: pointer;
-  margin: 0 $card-margin $base-spacing;
+  margin: 0 $card-margin $base-spacing $card-margin;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 $card-margin $base-spacing` instead of `0 $card-margin $base-spacing $card-margin`